includes/class-dispatcher.php

```diff
@@ -73,7 +79,55 @@ public static function process_outbox( $id ) {
 			$activity->set_object( $activity->get_object()->get_id() );
 		}
 
+		if ( ! self::should_send_to_followers( $activity, $actor_id, $outbox_item ) ) {
+			return;
+		}
+
 		self::send_activity_to_followers( $activity, $actor_id, $outbox_item );
+
+		// TODO: Should we call Scheduler::async_batch directly here?
+		\wp_schedule_single_event(
+			\time(),
+			'activitypub_async_batch',
+			array(
+				array( self::class, 'send_to_followers' ),
+				$activity,
+				$actor_id,
+				$outbox_item,
+				self::$batch_size,
+			)
+		);
+	}
+
+	/**
+	 * Asynchronously runs batch processing routines.
+	 *
+	 * @param Activity $activity    The ActivityPub Activity.
+	 * @param int      $actor_id    The actor ID.
+	 * @param \WP_Post $outbox_item The Outbox item.
+	 * @param int      $batch_size  Optional. The batch size. Default 50.
+	 * @param int      $offset      Optional. The offset. Default 0.
+	 * @return array|void The next batch of followers to process, or void if done.
+	 */
+	public static function send_to_followers( $activity, $actor_id, $outbox_item, $batch_size = 50, $offset = 0 ) {
+		$inboxes = Followers::get_inboxes_for_activity( $actor_id, $activity, $batch_size, $offset );
+		$json    = $activity->to_json();
+
+		foreach ( $inboxes as $inbox ) {
+			$result = safe_remote_post( $inbox, $json, $actor_id );
+
+			// @TODO Handle errors more elegantly.
+			if ( wp_remote_retrieve_response_code( $result ) >= 400 ) {
+				\error_log( 'Error sending to follower: ' . $inbox ); // phpcs:ignore
```

Comment:
no more storage of errors?

Reply:
See `// @TODO Handle errors more elegantly.` in line 119 :) 

But frankly, I'm not sure how we'll do that for Stream. Any ideas?

Reply:
should we simply call an action? this way we can use that for error processing.

* error log if WP_Debug
* stream if installed
* post-meta
* ...

Reply:
Sure! For each call individually or per batch?

Reply:
for each error?